### PR TITLE
Fix jasmine wrong test result

### PR DESCRIPTION
**Pull request by BorisOsipov**
Fix issue
https://github.com/webdriverio/wdio-allure-reporter/issues/94

### CODE REVIEW

File: test/specs/cucumber-test-case.js [hunk 1]
```diff
@@ -1,14 +1,43 @@
...
     it('should add feature & scenario labels for cucumber test cases', () => {
-        return runFeatures(['passing']).then((results) => {
+        return runCucumber(['passing']).then((results) => {
             const result = results[0]
             expect(result('test-case label[name="feature"]').eq(0).attr('value')).to.equal('A passing feature')
             expect(result('test-case label[name="story"]').eq(0).attr('value')).to.equal('A passing scenario')
         })
     })
+
+    it.only('should detect passed case', () => {
```
Comment (christian-bromann) [inline test/specs/cucumber-test-case.js]: .only

**Comment by christian-bromann** [comment]
Not sure why the cucumber test is failing but looks like a legit problem

**Comment by BorisOsipov** [comment]
@christian-bromann Thank you for careful review. I fixed "it.only" mistake and failed cucumber test.

**Comment by christian-bromann** [comment]
wdio-allure-reporter 0.4.0 published 🎉

**Pull request merged**